src/libambit/libambit.h

```diff
@@ -56,6 +56,7 @@ typedef struct ambit_personal_settings_s {
     uint8_t  sportmode_button_lock;
     uint8_t  timemode_button_lock;
     uint16_t compass_declination;
+    float compass_declination_f;      /* Ambit3: 32 bit float containing the radian */
```

Comment:
Could probably need to make the ambit2 convert the value to/from a float then, since uint16_t would not be large enough to represent the value from ambit3. I don't know what ambit2's declination value represent though. The ambit3 is simply a radian value (45 degrees == pi / 4) formatted as a IEEE 754 single precision float.